### ISSUE TITLE
Multiline TextField type numpad enter doesn't wrap

### ISSUE THREAD

**Issue by monkeyWie** [go-flutter-desktop/go-flutter#490]
When I type enter on numpad,it's doesn't wrap.
- ok:
![image](https://user-images.githubusercontent.com/13160176/87021409-c5f5c780-c207-11ea-994e-98cd343170ed.png)
- fail:
![image](https://user-images.githubusercontent.com/13160176/87021379-bc6c5f80-c207-11ea-8ba8-0ce0aa5d1d92.png)

## Image
![image](https://user-images.githubusercontent.com/13160176/87020844-04d74d80-c207-11ea-9237-8d7210ed858d.png)

## Hover doctor

```
$hover doctor
hover: Hover version v0.42.0 running on windows
hover: Sharing flutter version
Flutter 1.19.0-4.3.pre • channel beta • https://github.com/flutter/flutter.git
Framework • revision 8fe7655ed2 (8 days ago) • 2020-07-01 14:31:18 -0700
Engine • revision 9a28c3bcf4
Tools • Dart 2.9.0 (build 2.9.0-14.1.beta)
hover: Flutter engine commit: https://github.com/flutter/engine/commit/9a28c3bcf40ce64fee61e807ee3e1395fd6bd954
hover: Finding out the C compiler version
gcc (x86_64-posix-sjlj-rev0, Built by MinGW-W64 project) 8.1.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

hover: Sharing the content of go.mod
module text_demo/go

go 1.13

require (
        github.com/go-flutter-desktop/go-flutter v0.41.1
        github.com/pkg/errors v0.9.1
)

// replace github.com/go-flutter-desktop/go-flutter => /home/pchampio/lab/go/src/github.com/go-flutter-desktop/go-flutter
hover: Sharing the content of hover.yaml
application-name: ""
executable-name: ""
package-name: ""
license: ""
target: lib/main_desktop.dart
branch: ""
cache-path: ""
opengl: ""
engine-version: ""
hover: Sharing the content of go/cmd
go\cmd\main.go  go\cmd\options.go
```

**Comment by pchampio** [comment]
fixed in (`go-flutter 0.41.2`)[https://github.com/go-flutter-desktop/go-flutter/releases/tag/v0.41.2]

**Comment by monkeyWie** [comment]
Thanks, the fixed speed is amazing!
